crates/ironrdp-pdu/src/rdp/vc/dvc/display.rs

```diff
@@ -135,6 +136,8 @@ impl PduParsing for Monitor {
 
 /// [2.2.2.2] DISPLAYCONTROL_MONITOR_LAYOUT_PDU
 ///
+/// Deprecated in favor of the struct by the same name in crates/ironrdp-dvc.
```

Comment:
Let's remove those instead - eventually displaycontrol-related types in `ironrdp-dvc` should be removed too, and `ironrdp-displaycontrol` should be used instead.